frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -996,6 +1008,18 @@ https://archiveweb.page/guide`}
               const text = msg("Please enter a valid URL.");
               inputEl.helpText = text;
               inputEl.setCustomValidity(text);
+              inputEl.setCustomValidity(text);
```

Comment:
This appears to be a duplicate of the line above - maybe a mistake?

Reply:
yep, good catch!